### PR TITLE
docs: #775 Changed danger admonition in Console doc

### LINKED ISSUE #775: Console doc uses Danger admonition, which should not appear in docs
### Describe the Issue

(https://deploy-preview-773--redpanda-documentation.netlify.app/docs/platform/console/reference/config/)

Please change Dnager admonition to something else. Also, this would be a good opportunity to see if this is used anywhere else in Consle doc. Thx.

### PR DESCRIPTION
**Pull request by AnneLeeperRP**
Resolves #775 
Also did a quality edit, including comment text in config file.

### CODE REVIEW

**Comment by AnneLeeperRP** [comment]
@weeco 
Can you please review and check my edits for accuracy? Thank you!

**Review by weeco — APPROVED** [review]
LGTM, good reading improvements indeed :)

File: docs/platform/console/reference/config.mdx [hunk 1]
```diff
@@ -23,32 +23,30 @@ A reference YAML configuration file is provided under [Sample configuration](#sa
...
-All shown values reflect the default values. Provide the filepath to your configuration file by setting either the flag
+The following `config.yaml` configuration file contains a complete list of all Console configuration properties and their descriptions. All values are default values.
+
+Provide the filepath to your configuration file by setting either the flag
 `-config.filepath` or the environment variable `CONFIG_FILEPATH`.
 
 :::note
-This config file contains both enterprise and community configurations. If you don't provide an enterprise license, 
-Console ignores configurations for enterprise features.
+This configuration file contains both enterprise and community configurations. If you don't provide an enterprise license, Console ignores configurations for enterprise features.
```
Comment (Feediver1) [inline docs/platform/console/reference/config.mdx:49]:
```suggestion
This configuration file contains both Redpanda Enterprise and Redpanda Community configurations. If you don't provide a Redpanda Enterprise license, Console ignores configurations for Redpanda Enterprise features.
```

File: docs/platform/console/reference/config.mdx [hunk 2]
```diff
@@ -59,14 +57,14 @@ kafka:
...
-  # In multi zone Kafka clusters you can reduce traffic costs 
+  # In multi-zone Kafka clusters, you can reduce traffic costs 
   # by consuming messages from replica brokers in the same zone
   rackId:
   # SASL configuration for Console to connect to the Kafka API.
   sasl:
     enabled: false
     username:
-    # Password can be set via the --kafka.sasl.password flag as well.
+    # Password can also be set via the --kafka.sasl.password flag.
```
Comment (Feediver1) [inline docs/platform/console/reference/config.mdx:67]: per RP Style Guide, no "via":
# Password can also be set using the --kafka.sasl.password flag.

File: docs/platform/console/reference/config.mdx [hunk 3]
```diff
@@ -77,32 +75,30 @@ kafka:
       kerberosConfigPath:
       serviceName:
       username:
-      # Password can be set via the --kafka.sasl.gssapi.password flag as well
+      # Password can also be set via the --kafka.sasl.gssapi.password flag
```
Comment (Feediver1) [inline docs/platform/console/reference/config.mdx:78]:
```suggestion
      # Password can also be set using the --kafka.sasl.gssapi.password flag
```

File: docs/platform/console/reference/config.mdx [hunk 4]
```diff
@@ -77,32 +75,30 @@ kafka:
...
       password:
       realm:
       enableFast: true
     oauth:
-      # can be set via the --kafka.sasl.oauth.token flag as well
+      # can also be set via the --kafka.sasl.oauth.token flag
```
Comment (Feediver1) [inline docs/platform/console/reference/config.mdx:83]:
```suggestion
      # can also be set via the --kafka.sasl.oauth.token flag
```

Won't mark this moving forward--just do a global search and replace of "via". Thx

File: docs/platform/console/reference/config.mdx [hunk 5]
```diff
@@ -301,14 +295,14 @@ login:
 # can only be used with a valid Enterprise license.
 enterprise:
   rbac:
-    # Whether or not RBAC shall be used. This must be enabled
-    # if login is enabled. By default no authenticated user 
+    # Whether or not RBAC will be used. This must be enabled
```
Comment (Feediver1) [inline docs/platform/console/reference/config.mdx:298]:
```suggestion
    # Whether or not RBAC is used. This must be enabled
```

**Review by Feediver1 — APPROVED** [review]

**Pull request merged**